[ACM-12000] Updated uninstall topic for MCH (#6515)
open-cluster-management/rhacm-docs: install/uninstall.adoc

$ oc delete discoveryconfigs --all --all-namespaces
----

> * If you have agent service configs attached, you might see the following message:

Reviewer: ```suggestion
* If you have agent service configurations attached, you might see the following message:
```

+
[source,bash]
----
Cannot delete MultiClusterHub resource because AgentServiceConfig resource(s) exist
----
+

> - To disable and remove the `AgentServiceConfig` from by using the command line interface, complete the following steps:

Reviewer: ```suggestion
- To disable and remove the `AgentServiceConfig` resource by using the command line interface, complete the following steps:
```

$ oc delete discoveryconfigs --all --all-namespaces
----

> * If you have agent service configs attached, you might see the following message.

Reviewer: ```suggestion
* If you have agent service configs attached, you might see the following message:
```

+

> ----

Reviewer: ```suggestion
+
[source,bash]
----
```

Cannot delete MultiClusterHub resource because AgentServiceConfig resource(s) exist
----
+

> - To disable and remove the `AgentServiceConfig` using the terminal, see the following procedure:

Reviewer: ```suggestion
- To disable and remove the `AgentServiceConfig` from by using the command line interface, complete the following steps:
```

.. Log in to your hub cluster.

.. Delete the `AgentServiceConfig` custom resource by entering the following command:
+
----
$ oc delete agentserviceconfig --all

> ----

Reviewer: ```suggestion
.. Delete the `AgentServiceConfig` custom resource by entering the following command:
+
[source,bash]
----
$ oc delete agentserviceconfig --all
----
```

$ oc delete discoveryconfigs --all --all-namespaces
----

* If you have agent service configurations attached, you might see the following message:

+
[source,bash]
----
Cannot delete MultiClusterHub resource because AgentServiceConfig resource(s) exist
----
+
- To disable and remove the `AgentServiceConfig` resource by using the command line interface, complete the following steps:

> .. Log in to your hub cluster.

Reviewer 2: Why are these sub-steps, (..) ? 

Shouldn't it be one bullet ( .) since it's steps 1 and 2?

Reviewer 3: I will look at this, I didn't build the content, so there may be a reason or it could be oversight, but that is not on the engineer. This should come to me as the install writer. I have just been swamped. Let me look at the structure.

Reviewer 3: @dockerymick @jc-berger this content could be built better and is missing key UX standards, like anchors and descriptions under headers. For the sake of the release, we now have to edit for accuracy. Since this was created before we implemented peer reviews, it just has not been updated. This can be considered tech debt for next release.

Reviewer: @jc-berger @swopebe thanks to you both for the review. I had not examined the entire file. I just made some format updates. I hope that helps with moving forward in the next refresh.
